@types/express/index.d.ts

```
@@ -1,9 +1,23 @@
-declare namespace Express {
-  export interface Request {
-    responsePayload: Record<string, any>;
-    session: any;
+import 'express-session';
+
+declare module 'express-session' {
+  interface SessionData {
```

ESLint

[error] Insert `··`

(prettier/prettier)